CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does not yet adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 for setuptools_scm/PEP 440 reasons.
 
+
+## 2.2.0 Chia blockchain 2024-02-XX
+
+* Thanks to user https://github.com/bhorvitz for major help debugging a performance issue during coin DB lookup
```

Comment:
This is what we do in other cases, though none of them are getting linked by GitHub when viewing such as https://github.com/Chia-Network/chia-blockchain/blob/a64aafae6d4861b62c0b816face2c0e109aaea3c/CHANGELOG.md even though it creates links here in the diff.  `:[`

```suggestion
* Thanks to @bhorvitz for major help debugging a performance issue during coin DB lookup
```

I don't know what should be done here.  Maybe they should all be more like this.

```suggestion
* Thanks to [@bhorvitz](https://github.com/bhorvitz) for major help debugging a performance issue during coin DB lookup
```